.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on: 
+  push:
+    branches:
+      - '**'
+
+  # Allows you to run this workflow manually from the Actions tab
```

Comment:
sure we can leave the comment, but I guess once you are familiar with GH actions this becomes kind of obsolete.

Reply:
Agreed, removed in 419033e5